zipline-loader/src/commonMain/kotlin/app/cash/zipline/loader/ZiplineLoader.kt

```diff
@@ -308,40 +341,41 @@ class ZiplineLoader internal constructor(
 
     val loadedManifest = loadCachedOrEmbeddedManifest(applicationName, eventListener, now)
       ?: return null
+
     val startValue = eventListener.applicationLoadStart(applicationName, null)
     if (!freshnessChecker.isFresh(loadedManifest.manifest, loadedManifest.freshAtEpochMs)) {
       eventListener.applicationLoadSkippedNotFresh(applicationName, null, startValue)
       return null
-    } else {
-      try {
-        eventListener.manifestReady(applicationName, null, loadedManifest.manifest)
+    }
 
-        val zipline = loadFromManifest(
-          applicationName,
-          eventListener,
-          loadedManifest,
-          serializersModule,
-          now,
-          initializer,
-        )
-        eventListener.applicationLoadSuccess(
-          applicationName,
-          null,
-          loadedManifest.manifest,
-          zipline,
-          startValue,
-        )
-        send(LoadResult.Success(zipline, loadedManifest.manifest, loadedManifest.freshAtEpochMs))
-        return loadedManifest.manifest
-      } catch (e: CancellationException) {
-        // If emit() threw a CancellationException, consider that emit to be successful.
-        // That's 'cause loadOnce() accepts an element and then immediately cancels the flow.
-        throw e
-      } catch (e: Exception) {
-        send(LoadResult.Failure(e))
-        eventListener.applicationLoadFailed(applicationName, null, e, startValue)
-        return null
-      }
+    try {
```

Comment:
this isn’t actually changed; I just got IntelliJ to remove an unnecessary else clause.